syntax = "proto3";

service LoggingService {
    rpc LoggingService (stream LoggingRequest) returns (LoggingResponse);
}

message LoggingRequest {
    string message = 1;
    string timestamp = 2;
}

message LoggingResponse {
    string message = 1;
}
